app/controllers/openid_connect/user_info_controller.rb

```diff
@@ -0,0 +1,38 @@
+module OpenidConnect
+  class UserInfoController < ApplicationController
+    skip_before_action :verify_authenticity_token
+    before_action :authenticate_identity_via_bearer_token
+
+    attr_reader :current_identity
+
+    def show
+      render json: OpenidConnectUserInfoPresenter.new(current_identity).user_info
+    end
+
+    private
+
+    def authenticate_identity_via_bearer_token
```

Comment:
There's a lot of controller-specific control flow like rendering and stuff. WDYT about moving it to a mixin?